source/GameWindow.cpp

```diff
@@ -363,7 +363,8 @@ void GameWindow::ExitWithError(const string& message)
 	box.buttons = &button;
 	
 	int result = 0;
-	SDL_ShowMessageBox(&box, &result);
+	if(doPopUp)
```

Comment:
A lot of the other code in this method should be skipped if we aren't displaying an error box to the user.